fastly-compute/geoip/.github/workflows/deploy-geoip.yaml

```diff
@@ -0,0 +1,18 @@
+name: Deploy Compute@Edge GeoIp
```

Comment:
this file needs to move to `.github/workflows/deploy-geoip.yaml` to be triggered.